Missing ManagementPointNetworkType param (#2531)
MicrosoftDocs/windows-powershell-docs: docset/winserver2019-ps/failoverclusters/New-Cluster.md

```
New-Cluster [-Name] <String> [-Node <StringCollection>] [-StaticAddress <StringCollection>]
 [-IgnoreNetwork <StringCollection>] [-NoStorage] [-S2D] [-AdministrativeAccessPoint <AdminAccessPoint>]

>  [-Force] [-ManagementPointNetworkType {Automatic | Singleton | Distributed}] [<CommonParameters>]

Reviewer: ```suggestion
 [-Force] [-ManagementPointNetworkType <AdminAccessPointResType>] [<CommonParameters>]
```

Reviewer 2: At least as far as I can find examining the cmdlet in PowerShell, the defined parameter type is `object` so I don't think this change would be correct.

```powershell
(Get-Command New-Cluster).Parameters.ManagementPointNetworkType.ParameterType

IsPublic IsSerial Name                                     BaseType                                                     
-------- -------- ----                                     --------
True     True     Object                                                                                                                                                                             
```

Reviewer: The parameter type is indeed AdminAccessPointResType according to PlatyPs:

![image](https://user-images.githubusercontent.com/36666927/126366362-2bb44afe-fcbb-48fc-9df2-c7fd79ad5b37.png)

Reviewer: Let me know what you think @chasewilson  but we usually honor PlatyPs

Reviewer 2: Yup usually the best thing to do. Good by me

Accept wildcard characters: False
```

### -ManagementPointNetworkType
Specifies the network configuration used to determine IP address settings.

The acceptable values for this parameter are:

> - Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`. (Default Setting)

Reviewer 3: ```suggestion
- Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`. (Default setting.)
```

- Singleton: The traditional method of DHCP or static IP address.

> - Distributed: Use a Distributed Network Name by using Node IP addresses.

Reviewer 3: ```suggestion
- Distributed: Uses a Distributed Network Name by using Node IP addresses.
```

Accept wildcard characters: False
```

### -ManagementPointNetworkType
Specifies the network configuration used to determine IP address settings.

The acceptable values for this parameter are:

> ```yaml

Reviewer 2: I would suggest against using a yaml block to represent the options. I would follow something more like the `AdministrativeAccessPoint` template and use  a bulleted list.

Reviewer 4: I also agree with removing the yaml block.

> Singleton: Uses the traditional method of DHCP or static IP address.

Reviewer 2: ```suggestion
Singleton: The traditional method of DHCP or static IP address. (Default setting)
```

Distributed: Use a Distributed Network Name by using Node IP addresses.

> Automatic: Uses detection to determine the appropriate setting. If SQL Server is running in Azure, uses Distributed. If SQL Server is running on-premises, uses Singleton (default setting).

Reviewer 2: ```suggestion
Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`.
```

```

```yaml
Type: AdminAccessPointResType
Parameter Sets: (All)
Aliases: 

Required: False
Position: Named

> Default value: None

Reviewer 2: ```suggestion
Default value: Singleton
```

Double check this information to ensure accuracy. I'm making this changes based on your changes in the list of values above.

Reviewer 4: When I used PlatyPS for WS2022, the default value was None.

Author: I believe that Automatic is the default value for this parameter, according to the few sources I've seen.
